### PR TITLE
Added import statements to examples in documentation

### LINKED ISSUE #593: Add import statements to examples in documentation
See title.

The examples in the documentation should also include the `from manim import *` at the very least, and actually we could provide best-practice examples where we dont do a *-import, but rather import classes/functions separately.

This can of course be an iterative process: start with adding `from manim import *` first, and become more specific later.

### PR DESCRIPTION
**Pull request by Darylgolden**
## Motivation
Closes #593, there is consensus to add import statements to the start of every code example.

## Explanation for Changes
Added the import statement in the manim directive.

## Testing Status
Let's see if the docs builds correctly in the CI.

## Checklist
- [x] I have read the [Contributing Guidelines](https://docs.manim.community/en/latest/contributing.html)
- [x] I have written a descriptive PR title (see top of PR template for examples)
- [x] I have written a changelog entry for the PR or deem it unnecessary
- [ ] My new functions/classes either have a docstring or are private
- [ ] My new functions/classes have [tests](https://github.com/ManimCommunity/manim/wiki/Testing) added and (optional) examples in the docs
- [x] My new documentation builds, looks correctly formatted, and adds no additional build warnings

## Reviewer Checklist
- [ ] The PR title is descriptive enough
- [ ] The PR is labeled correctly
- [ ] The changelog entry is completed if necessary
- [ ] Newly added functions/classes either have a docstring or are private
- [ ] Newly added functions/classes have [tests](https://github.com/ManimCommunity/manim/wiki/Testing) added and (optional) examples in the docs
- [ ] Newly added documentation builds, looks correctly formatted, and adds no additional build warnings

### CODE REVIEW

File: docs/source/manim_directive.py [hunk 1]
```diff
@@ -202,6 +202,7 @@ def run(self):
         source_block = [
             ".. code-block:: python",
             "",
+            "from manim import *\n\n",
```
Comment (jsonvillanueva) [inline docs/source/manim_directive.py:205]: Very clever! I think this just needs proper indentation and it's good to go.

**Review by jsonvillanueva — APPROVED** [review]

**Comment by kolibril13** [comment]
image
Looks good!
One can see that in the `manim` at the import line has an underscore in the docs.
Is this intended? It looks like it is a hyperlink, but then it is not clickable.

**Review by kolibril13 — APPROVED** [review]

**Comment by Darylgolden** [comment]
> 
> 
> image
> Looks good!
> One can see that in the `manim` at the import line has an underscore in the docs.
> Is this intended? It looks like it is a hyperlink, but then it is not clickable.

It's not intended, and I also don't see the underscore on my browser, what browser are you using?

**Comment by kolibril13** [comment]
> It's not intended, and I also don't see the underscore on my browser, what browser are you using?

I am using google chrome.
Let me test it with another browser.

**Comment by kolibril13** [comment]
I can see the underscore in google chrome and safari, but only in dark mode.
There is no underscore in bright mode!

**Comment by Darylgolden** [comment]
I see it too now. Not sure why though, it probably has something to do with the theme.

**Comment by kolibril13** [comment]
Ok, we can merge it nevertheless and address this in a future pr.

**Comment by behackl** [comment]
Something like 

```css
    span.nc, span.nn {
        text-decoration: none !important;
    }
```

has to be changed in the `@media (prefers-color-scheme: dark)` block in `custom.css` to get rid of the underline.

**Comment by jsonvillanueva** [comment]
Modifications to the theme can be done in a different PR

**Pull request merged**